Dockerfile

```diff
@@ -1,5 +1,6 @@
-FROM continuumio/anaconda3
+FROM continuumio/miniconda3:4.6.14
```

Comment:
@lwasser the main difference is here. We are better off with a light miniconda rather than the full Anaconda distro.